### DISCUSSION TITLE
【委托】想知道同时吃“安眠药，春药，泻药，止泻药”和饭会怎么样

### DISCUSSION THREAD

**Discussion by 1luik** [share121/inter-knot#3126] [其他]
![68747470733a2f2f75706c6f61642d6262732e6d69796f757368652e636f6d2f75](https://github.com/user-attachments/assets/eae3189a-0d21-4ab4-8ea8-6834d5faa1e7)
委托费一个人1000万

**Comment by opqphp** [comment]
会做*梦吗<img src="https://bbs-static.miyoushe.com/static/2024/06/19/a751ee1e27d4253ea0c233c905f36085_3464665030406935104.png" alt="托帕 眺望" width="75" >

**Comment by DCQ07** [comment]
会进医院

**Reply by 1luik** [reply]
如果你能抗就不需要，钟离先生你可以用你的盾试试吗

**Reply by DCQ07** [reply]
我觉得这应该是直接扣血量，而不是扣护盾了，流血狗防不住啊

**Comment by RaphaelHider** [comment]
我去试试，等我

**Reply by haxyyx** [reply]
怎么样了

**Reply by 1luik** [reply]
她可能睡着了

**Reply by DCQ07** [reply]
或许还在买药的路上，谁家里常备这些药啊

**Reply by haxyyx** [reply]
说不定已经到当地的医院了吧，希望人没事，不会第二天上绳网推荐吧

**Reply by Pray-Fay** [reply]
不是，来真的啊？要注意自己身体

**Reply by mochengyanliu920** [reply]
注意身体

**Comment by 56712354** [comment]
这个委托我接了。
现在身体像尸体一样冰凉。
头一回感觉到世界这么温暖。

**Comment by writhlly** [comment]
会梦到冈交吧

**Comment by Anyi-qaq** [comment]
春药和安眠药一起吃可能会导致神经系统紊乱，表现为意识模糊，极度嗜睡或兴奋，幻觉，癫痫和喘不上气。
泻药和止泻药一起吃可能会导致肠道痉挛，进而导致剧烈腹泻或肠梗阻。
并且，若泻药和止泻药中含有阿片类成分会进一步加重呼吸抑制效果，造成缺氧。
与食物同服会加重肠道负担，最终造成想上吐下泻的后果

**Comment by Kazuto-Kiritani** [comment]
fairy(DeepSeek)说
![1](https://github.com/user-attachments/assets/b35a65b5-f50a-4d73-b74d-b03d5d30300b)
![2](https://github.com/user-attachments/assets/ab1a8dd2-6ffc-4382-9c26-bc9e7e2e94fe)
![3](https://github.com/user-attachments/assets/fec80bf8-de3e-41fb-9e4b-ceb3f5c035fb)

**Comment by blqdswkl** [comment]
会死掉（简单明了）

**Comment by fengganchun** [comment]
肝脏会爆炸

**Comment by Onerix** [comment]
会上绳网

**Comment by xudaz** [comment]
会发生身体紊乱，有各种负面效果。且喝个牛奶能不能解毒!

**Comment by spoopop** [comment]
那很会吃了

**Comment by spoopop** [comment]
![155B34BDA12C3E76D91878FCB27F4EA1](https://github.com/user-attachments/assets/8a4a9b18-a8b9-4d19-a141-f1ddbf8c751d)
